pkgs/applications/version-management/git-and-tools/default.nix

```diff
@@ -127,6 +127,8 @@ let
 
   gitflow = callPackage ./gitflow { };
 
+  gitflow = callPackage ./gitstatus { };
```

Comment:
gitstatus I think :p

Reply:
derp :)

Reply:
Thats what I get for not cherry-picking from my other branch :|

Reply:
fixed